spec/integration/users_spec.js

```diff
@@ -0,0 +1,251 @@
+const request = require("request");
+const server = require("../../src/server");
+const base = "http://localhost:8000/users/";
+const accountBase = "http://localhost:8000/account/";
+
+const User = require("../../src/db/models").User;
+const sequelize = require("../../src/db/models/index").sequelize;
+const stripe = require("stripe")(process.env.STRIPE_API_KEY);
```

Comment:
this is good! never hardcode kind of credential or private key in a source file (or in any other file that gets committed to your repo)